code/controllers/subsystem/triumph/buy_datums/character/leprosy.dm

```diff
@@ -1,16 +1,16 @@
 /datum/triumph_buy/leprosy
 	name = "Leprosy"
-	desc = "Become a leper. You will be hated, you will be shunned, you will bleed and you will be weak. But Pestra will be always on your side."
+	desc = "Become a leper. You will be hated, you will be shunned, you will bleed and you will be weak. But Pestra will take all your pain away."
 	triumph_buy_id = TRIUMPH_BUY_LEPROSY
-	triumph_cost = 3
+	triumph_cost = 2
 	category = TRIUMPH_CAT_CHARACTER
 	visible_on_active_menu = TRUE
 	manual_activation = TRUE
-	allow_multiple_buys = FALSE
 
 /datum/triumph_buy/leprosy/on_after_spawn(mob/living/carbon/human/H)
 	. = ..()
 	ADD_TRAIT(H, TRAIT_LEPROSY, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_NOPAIN, TRAIT_GENERIC)
```

Comment:
this is a flavor trait do not give them mechanical advantages

Reply:
Its not a flavor trait, its effects are very real and scale is massively against them even with this